### DISCUSSION TITLE
启动服务后没法联网

### DISCUSSION THREAD

**Discussion by koalaaaaaaaaa** [v2fly/v2ray-core#2892] [Q&A]
首先感谢大佬的解答[抱歉][抱拳]祝大佬工作顺利赚大钱！！
系统：linux ubuntu
描述：服务看起来正常启动了，但是没法连外网。服务启动后的截图如下：
![image](https://github.com/v2fly/v2ray-core/assets/59307253/336cbbf7-83a7-4486-b581-7818338df37e)
我的config文件截图如下：
![image](https://github.com/v2fly/v2ray-core/assets/59307253/279dd832-0064-49a0-bf43-822d7f2a76c5)
![image](https://github.com/v2fly/v2ray-core/assets/59307253/f38cd06a-e24f-4310-8830-048f6c97b748)
![image](https://github.com/v2fly/v2ray-core/assets/59307253/a8904af4-e0af-448a-b30e-bee372b9fa6e)
其中vnext里面的服务器地址和端口是从下图这边复制过去的。
![image](https://github.com/v2fly/v2ray-core/assets/59307253/f161b12f-70e9-4d13-9d9e-6fa063bac3b8)

**Comment by tomandjerry27** [comment]
从你的描述来看，你好像不知道config.json至少有两个，一个在服务器上，一个在客户机上。也就是说你连自己的情况都没有讲清楚。
